enums.go

```diff
@@ -0,0 +1,282 @@
+package openrtb
```

Comment:
Thank you for this, but, please, **remove it entirely**.

I won't accept enums into this repo yet. Either keep these enums in your own lib, or fork this repo and switch your code to use it.

I'll leave some style-critics on this file anyway, but don't fix them (they're more like "friendly advices") - just remove this file from PR.